packages/json-api/src/Response/responseNormalizer.js

```diff
@@ -53,6 +53,9 @@ function responseNormalizer(response) {
   const rootResources = normalizeResources(singleMode ? [response.data] : response.data);
   relationshipApplyToData(rootResources, includedResources);
 
+  Object.values(includedResources).forEach(resource => resource.resetStateStack());
+  Object.values(rootResources).forEach(resource => resource.resetStateStack());
+
```

Comment:
Ik twijfel of we dit niet nog even in een losse functie moeten zetten. Tot nu toe is deze hele functie redelijk vrij van eigen logica. Met een functie maken we extra duidelijk wat er gebeurd. Kan gewoon een losse functie in deze file zijn, hoeft niet geexport te worden.

In de naam de reden waarom het gebeurd?
ensureResourceSingleStateInStack?

Reply:
Zo changed :)